lib/matplotlib/mpl-data/stylelib/classic.mplstyle

```diff
@@ -497,3 +497,5 @@ animation.convert_path: convert   # Path to ImageMagick's convert binary.
                                   # is also the name of a system tool.
 animation.convert_args:
 animation.html: none
+
+_internal.classic_mode: True
```

Comment:
PEP8 doesn't apply as this isn't a Python file ;)  But yes, we can go in and add the newline next time we touch this file.